conf/bafu.conf.part

```diff
@@ -132,6 +132,74 @@ source src_ch_bafu_hydrologie-untersuchungsgebiete_stationen : def_searchable_fe
         from hydrologie.hydrologie_untersuchungsgebiete_stationen
 }
 
+source src_ch_bafu_hydrologie-daueruntersuchung_fliessgewaesser : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT objectid as id \
+            ,  name as label \
+            , 'feature' as origin \
+            ,  remove_accents(name)||' '||remove_accents(betriebsbeginn)||' '||remove_accents(stationierung)||' '||remove_accents(flussgebiet)||' '||remove_accents(einzugsgebietsflaeche)||' '||remove_accents(mittlerehoehe)||' '||remove_accents(hoehe)||' '||remove_accents(vergletscherung) as detail \
```

Comment:
concatenated attributes have to be wrapped inside coalesce function (as described in wiki)
so please replae
```sql
remove_accents(name)||' '||...
-- with
remove_accents(coalesce(name,' '))||' '||..
```